script/configs/exclude_integration_ios.yaml

```diff
@@ -1,7 +1,5 @@
-# Currently missing: https://github.com/flutter/flutter/issues/81695
-- in_app_purchase_storekit
```

Comment:
Since there's a lot in this PR (even if a lot of it is fairly boilerplate), could you extract this integration test fix to a small separate PR that we could land quickly while the rest is being reviewed? That would make iteration on the rest easier.

Reply:
Yep, will do that.

Reply:
Integration test fix is now in https://github.com/flutter/plugins/pull/6604.